### PR TITLE
adapter: add debug option

### PR DESCRIPTION
**Pull request by xhaakon**
Prints RDK messages to stdout. Run as

  cargo run -- -d my-rdk.ddns.net --debug=true

### CODE REVIEW

**Review by bgy36ww — APPROVED** [review]

**Comment by xhaakon** [comment]
@bgy36ww Thanks for approval. In case you're waiting for me to finish the merge, I need write access to the repo first.

**Pull request merged**
